python/ray/rllib/agents/dqn/torch_dqn_policy_graph.py

```diff
@@ -0,0 +1,7 @@
+from __future__ import absolute_import
+
+from ray.rllib.evaluation.torch_policy_graph import TorchPolicyGraph
+
+
+class DQNTorchPolicyGraph(TorchPolicyGraph):
```

Comment:
just fyi, another alternative is to extend PolicyGraph directly, as in https://github.com/ray-project/ray/blob/master/python/ray/rllib/agents/qmix/qmix_policy_graph.py

This could end up make it easier to implement.

Reply:
Thanks! I'll have a look

Reply:
I guess also QMIX is technically also a DQN :) Albeit extremely unoptimized for single-agent cases.